### PR TITLE
feat: add attributionClass to MyCollectionCreateArtworkInput

### PR DESCRIPTION
**Pull request by fladson**
This PR resolves to add attribution Class in MyCollection artwork - [CX-2224](https://artsyproduct.atlassian.net/browse/CX-2224)

### CODE REVIEW

**Review by olerichter00 — CHANGES_REQUESTED** [review]
Looking good! 🙌 

Could you please add a description to the PR (so we don't rely on third-party services like Jira to find out what this change is about) and have a look if we can update the corresponding test files as well?

File: _schemaV2.graphql [hunk 1]
```diff
@@ -1607,6 +1607,13 @@ enum ArtworkAggregation {
   TOTAL
 }
 
+enum ArtworkAttributionClassType {
```
Comment (fladson) [inline _schemaV2.graphql:1610]: Is `ConsignmentAttributionClass` referring to the same thing? Just wondering if we could unify it into only one enum.

Comment (olerichter00) [reply _schemaV2.graphql:1610]: That would be great but I guess `ConsignmentAttributionClass` is not an input type, right?

**Review by olerichter00 — APPROVED** [review]
🚀 🚀 🚀

**Pull request merged**